umbral/fragments.py

```diff
@@ -57,7 +73,15 @@ def expected_bytes_length(cls, curve: Optional[EllipticCurve] = None) -> int:
         bn_size = CurveBN.expected_bytes_length(curve)
         point_size = Point.expected_bytes_length(curve)
 
-        return (bn_size * 4) + (point_size * 3)
+        # self.id --> 1 bn_size
+        # self._bn_key --> 1 bn_size
+        # self._point_commitment --> 1 point_size
+        # self._point_precursor --> 1 point_size
+        # self.signature_for_proxy --> 2 bn_size
+        # self.signature_for_bob --> 2 bn_size
+        # self.keys_in_signature --> 1
```

Comment:
I think this is overly-verbose, but should definitely have a comment for `keys_in_signature`. The `bn_size` and `point_size` stuff is pretty self-evident in the `return`.

Reply:
Well, apart from `keys_in_signature`, the size of `self.id` is arbitrary too, since it's a byte string. I agree that it's a big comment, but in this case I think it's justified.